test/gtest/uct/test_md.cc

```diff
@@ -841,6 +841,7 @@ UCS_TEST_SKIP_COND_P(test_md, exported_mkey,
     dereg_params.memh       = export_memh;
     status                  = uct_md_mem_dereg_v2(md(), &dereg_params);
     ASSERT_UCS_OK(status);
+    free(ptr);
```

Comment:
IMO better make ptr an std::vector/string so it will be release also when test fails by ASSERT_UCS_OK

Reply:
yes, better to use C++ allocation but for other reason - proper handling of memory allocation and code style alignment. Failed `ASSERT_UCS_OK` should kill the process, so no need to care about `ptr` release.